content/minfraud/api-documentation/responses.mdx

```diff
@@ -0,0 +1,229 @@
+---
+draft: false
+title: minFraud API Responses
+---
+
+import * as responseJson from './_examples/response'
+import * as Schemas from './_schemas/';
+
+## Headers
+
+<table>
+  <thead>
+    <tr>
+      <th>Header</th>
+      <th>Notes</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        `Content-Type`
+      </td>
+      <td>
+        The `Content-Type` header for a successful response varies based on the
+        service as outlined above.
+
+        Errors may be returned with the `Content-Type` set to
+        `application/vnd.maxmind.com-error+json; charset=UTF-8; version=2.0`.
+        If this is the case, then the body of the response contains a JSON
+        document with two keys, code and error. See the [Errors](#) section for
+        more details.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `Content-Length`
+      </td>
+      <td>
+        The response will always include a `Content-Length` header as well.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Bodies
+
+Each service returns data as a JSON document. The document that is returned
+always consists of an object (aka map or hash). Below are the schema definitions
+that describe each service's response body.
+
+### Score Body
+
+<Example language="json">
+  {JSON.stringify(responseJson.score, null, 2)}
+</Example>
+
+### Insights Body
+<Example language="json">
+  {JSON.stringify(responseJson.insights, null, 2)}
+</Example>
+
+### Factors Body
```

Comment:
the order of objects is different than the Factors example on https://dev.maxmind.com/minfraud/#Response_Body_Examples

Reply:
See previous comment about alphasorting of properties.